### PR TITLE
Hw 05

### PR DESCRIPTION
**Pull request by AnnetTumani**

### CODE REVIEW

**Review by AnnetTumani — COMMENTED** [review]
Vadim Papko

File: hw_05/Task2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                mass[i] = mass[i-1];
+            }
+            mass[poss-1] = x;
+
+            Console.WriteLine("Полученный массив: ");
+            foreach (int element in mass)
+            {
+                Console.Write(element + " ");
+            }
+
```
Comment (VadzimPapko) [inline hw_05/Task2/Program.cs:45]: just remove one empty line there

File: hw_05/Task3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            int[] array = new int[elementsCount];
+            var randomizer = new Random();
+            for (int i = 0; i < array.Length; i++)
+            {
+                array[i] = randomizer.Next(1, 25);
+            }
+            return array;
+        }
+    }
+}
```
Comment (VadzimPapko) [inline hw_05/Task3/Program.cs:53]: well done

**Review by VadzimPapko — APPROVED** [review]
reviewed

**Comment by VadzimPapko** [comment]
I see many files that do not need to add into GIT. Such as, *.exe, *.pdb, .etc. Please use .gitignore file to exclude files from GIT

**Pull request merged**